src/ir/enum_ty.rs

```diff
@@ -71,16 +71,20 @@ impl Enum {
         };
 
         declaration.visit(|cursor| {
-            if cursor.kind() == CXCursor_EnumConstantDecl {
```

Comment:
why not keeping the `if` above as before, and then just `unwrap` the result of `enum_val_signed`?